### ISSUE TITLE
Permalink issue

### ISSUE THREAD

**Issue by dpvtank** [maximevaillancourt/digital-garden-jekyll-template#3]
Apologies if this is not the right place to do this since I'm not very tech literate and not very good at Github.

I forked this repo, and linked it to my personal website as a project. I try to access it at dtank.co/digital-garden

but the links don't work. I changed the permalink section in the config file, but to no avail.

Any help?

**Comment by dpvtank** [comment]
Figured out how to do it over the last 2 days.

**Comment by maximevaillancourt** [comment]
Hey Dhawal, thanks for the update. If you're willing to share how you fixed your issue, it may help other people facing the same problem! :pray:

**Comment by dpvtank** [comment]
Sure. Here's what I did:
1. create a CNAME record in the folder linking directly to the website I wanted. In this case, it was mind.dtank.co
2. Go into DNS records and create a CNAME record with my domain provider and pointed it to my main github page. In this case, it was dpvtank.github.io
3. Go into settings of the repo, and select push repo to master. And then under custom domain, I put in mind.dtank.co.

This helped me create the right subdomain and fixed the permalink issues.

Before however, when I did not create a subdomain and had everything running on dtank.co/digital-garden, the links on each page would go to the main site (e.g. dtank.co), not dtank.co/digital-garden/whatever-note
